### DISCUSSION TITLE
Prisma scripts fail to download engine on fresh install ECONNRESET

### DISCUSSION THREAD

**Discussion by ZacharyEggert** [prisma/prisma#18956] [Q&A] [answered]
### Bug description

Prisma script stalls at engine download

```
> Downloading Prisma engines for Node-API for windows [                    ] 0%

Error: request to https://binaries.prisma.sh/all_commits/1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a/windows/migration-engine.exe.gz.sha256 failed, reason: read ECONNRESET

error Command failed with exit code 1.
```

### How to reproduce

In an empty directory, on a Windows 11 Machine

```code

> yarn init -y

> yarn add @prisma/client prisma

> yarn prisma init

```

*the issue is the same when using npm instead of yarn*

### Expected behavior

Should download the appropriate engine for the platform, and create a template file in a new subdirectory in the cwd.

### Prisma information

N/A

### Environment & setup

- OS: Windows 11 
- Database: N/A 
- Node.js version: 19.9.0 / 20.0.0 

### Prisma Version

```
4.13.0
```

**Comment by janpio** [comment]
Can you download https://binaries.prisma.sh/all_commits/1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a/windows/migration-engine.exe.gz.sha256 in your browser? That file seems to work perfectly fine to me, so this is probably something in your local setup.

**Comment by ZacharyEggert** [comment, marked as answer]
I changed networks and it seems to work fine now, so it looks like I'll have to do some troubleshooting on my network's end of things
